clients/unshaded/src/test/java/tachyon/client/file/options/CompleteFileOptionsTest.java

```diff
@@ -20,36 +20,28 @@
 import org.junit.Assert;
 import org.junit.Test;
 
-import tachyon.conf.TachyonConf;
-
 /**
  * Tests for the {@link CompleteFileOptions} class.
  */
 public class CompleteFileOptionsTest {
-
   /**
-   * Tests that building a {@link CompleteFileOptions} works.
+   * Tests that building a {@link CompleteFileOptions} with the defaults works.
    */
   @Test
-  public void builderTest() {
-    Random random = new Random();
-    long ufsLength = random.nextLong();
-
-    CompleteFileOptions options =
-        new CompleteFileOptions.Builder(new TachyonConf())
-            .setUfsLength(ufsLength)
-            .build();
+  public void defaultsTest() {
+    CompleteFileOptions options = CompleteFileOptions.defaults();
 
-    Assert.assertEquals(ufsLength, options.getUfsLength());
+    Assert.assertEquals(0, options.getUfsLength());
   }
 
   /**
-   * Tests that building a {@link CompleteFileOptions} with the defaults works.
+   * Tests getting and setting fields
```

Comment:
Sure, updated. On a side note, this seems unreasonable to enforce without a checkstyle. 
/cc @jsimsa